### PR TITLE
Docker updates

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
- @kiritigowda Removes  `--usecase=graphics` from all dockerfiles as mentioned in  issue [#147](https://github.com/ROCm/rocAL/issues/147)
- Updates some old libraries 
- removes cupy from pytorch docker, since it doesn't depend on it

### CODE REVIEW

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install initramfs-tools libnuma-de
         wget ${ROCM_INSTALLER_REPO} && \
         sudo apt-get install -y ./${ROCM_INSTALLER_PACKAGE} && \
         sudo apt-get update -y && \
-        sudo amdgpu-install -y --usecase=graphics,rocm --no-32
+        sudo amdgpu-install -y --usecase=rocm
```
Comment (rrawther) [inline docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile:23]: do we work with ffmpeg HW acceleration when we remove this usecase

**Review by rrawther — CHANGES_REQUESTED** [review]
Added a comment

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile (hunk 1 above)
Comment (LakshmiKumar23) [reply docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile:23]: We do not have the graphics usecase going forward from rocm6.2. @kiritigowda please correct me if I am wrong here. We need to install the dependencies like we do for rocDecode I think

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile [hunk 2]
```diff
@@ -51,17 +51,12 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install wget libbz2-dev libssl-dev
         cmake -DBACKEND=HIP ../ && make -j4 && sudo make install && cd ../../ && \
         git clone -b v3.12.4 https://github.com/protocolbuffers/protobuf.git && cd protobuf && git submodule update --init --recursive && \
         ./autogen.sh && ./configure && make -j8 && make check -j8 && sudo make install && sudo ldconfig && cd
-ENV CUPY_INSTALL_USE_HIP=1
-ENV ROCM_HOME=/opt/rocm
+
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python3 python3-pip git g++ hipblas hipsparse rocrand hipfft rocfft rocthrust-dev hipcub-dev python3-dev && \
         git clone https://github.com/Tencent/rapidjson.git && cd rapidjson && mkdir build && cd build && \
         cmake ../ && make -j4 && sudo make install && cd ../../ && \
         pip install pytest==3.1 && git clone -b v2.10.4 https://github.com/pybind/pybind11 && cd pybind11 && mkdir build && cd build && \
```
Comment (SundarRajan98) [inline docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile:58]: pytest package version should be 7.3.1 (released Apr 2023). This version is released on May 2017. Applies for all dockerfiles

**Comment by kiritigowda** [comment]
@LakshmiKumar23 -- let me add the va-drivers link to TOT develop and I will merge this change 

Issue #147

File: docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile (hunk 2 above)
Comment (LakshmiKumar23) [reply docker/rocal-on-ubuntu-20-with-pytorch-with-mesa.dockerfile]: Done

**Comment by kiritigowda** [comment]
@LakshmiKumar23 can you please resolve conflicts

**Comment by LakshmiKumar23** [comment]
> @LakshmiKumar23 can you please resolve conflicts

@kiritigowda done, can you check again?

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**